File: PB - More Exercises Python/3.0 Nested Conditional Statements - More Exercises/04. Car To Go.py
budget, season = float(input()), input()
class_type, car_type = '', ''

if budget <= 100:
    class_type = 'Economy class'
    if season == 'Summer':
        budget *= 0.35
        car_type = 'Cabrio'
    else:
        budget *= 0.65
        car_type = 'Jeep'
elif 100 < budget <= 500:
    class_type = 'Compact class'
    if season == 'Summer':
        budget *= 0.45
        car_type = 'Cabrio'
    else:
        budget *= 0.80
        car_type = 'Jeep'
else:
    class_type = 'Luxury class'
    budget *= 0.90
    car_type = 'Jeep'

print(f'{class_type}\n{car_type} - {budget:.2f}')


'''
70.50
Winter
'''

File: PB - More Exercises GO/3.0 Nested Conditional Statements - More Exercises/04. Car To Go.go
package main

import "fmt"

func main() {
	var budget float64
	var season, classType, carType string
	fmt.Scan(&budget, &season)

	if budget <= 100 {
		classType = "Economy class"
		if season == "Summer" {
			budget *= 0.35
			carType = "Cabrio"
		} else {
			budget *= 0.65
			carType = "Jeep"
		}
	} else if 100 < budget && budget <= 500 {
		classType = "Compact class"
		if season == "Summer" {
			budget *= 0.45
			carType = "Cabrio"
		} else {
			budget *= 0.80
			carType = "Jeep"
		}
	} else {
		classType = "Luxury class"
		budget *= 0.90
		carType = "Jeep"
	}

	fmt.Printf("%v\n%v - %.2f", classType, carType, budget)
}

/*
name   :04. Car To Go
input  :450 Summer
output :
Compact class
Cabrio - 202.50
*/
